pkgs/by-name/tr/trdsql/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "trdsql";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "noborus";
+    repo = "trdsql";
+    tag = "v${version}";
+    hash = "sha256-MkjQAOIXnydEmOFnnYrvE2TF2I0GqSrSRUAjd+/hHwc=";
+  };
+
+  vendorHash = "sha256-PoIa58vdDPYGL9mjEeudRYqPfvvr3W+fX5c+NgRIoLg=";
+
+  doCheck = false; # panic: open /etc/protocols: operation not permitted
```

Comment:
Can't reproduce this error on `x86_64-linux`

Reply:
It can reproduce in darwin.
Do you have a idea about how to fix it?

Reply:
Not really, sorry. You could however disable the checks on darwin only using something like `doCheck = !stdenv.hostPlatform.isDarwin;`